report.py

```diff
@@ -425,16 +415,20 @@ def _parse_args() -> Any:
         _color_percent(statistics_table[MarketAttribute.ROI][MarketState.CLOSED]),
     )
 
-    since_ts = time.time() - 60 * 60 * 24 * TRADES_LOOKBACK_DAYS
+    since_ts = time.time() - DAY_IN_UNIX * TRADES_LOOKBACK_DAYS
     _print_status(
         f"Trades on last {TRADES_LOOKBACK_DAYS} days",
         _trades_since_message(trades_json, since_ts),
     )
 
-    #Multi bet strategy
-    _print_subsection_header(f"Markets with multiple bets in previous {REBETS_LOOKBACK_DAYS} days")
-    rebets_since_ts = time.time() - 60 * 60 * 24 * REBETS_LOOKBACK_DAYS
-    _print_status(f"Multi-bet markets", _rebets_since_message(BETS_JSON_PATH, trades_json, rebets_since_ts))
+    try:
+        #Multi bet strategy
+        _print_subsection_header(f"Markets with multiple bets in previous {REBETS_LOOKBACK_DAYS} days")
+        rebets_since_ts = time.time() - DAY_IN_UNIX * REBETS_LOOKBACK_DAYS
+        _print_status(f"Multi-bet markets", _rebets_since_message(trades_json, rebets_since_ts))
+
+    except Exception:
+        print("An error occurred while calculating rebets.")
```

Comment:
This would be hard to debug. Maybe show the stack trace too? In general, I would suggest avoiding this general try catch and instead try to make the inner code of these two methods as robust as possible.